tests/src/Simple/ReadyToRunUnit/Program.cs

```diff
@@ -61,6 +71,23 @@ private static bool IsInstanceOfValueType()
         }
     }
     
+    private unsafe static bool CheckNonGCThreadLocalStatic()
+    {
+        fixed (int *lineCountPtr = &LineCount)
+        {
+            Console.WriteLine($@"LineCount: 0x{LineCount:X8}, @ = 0x{(ulong)lineCountPtr:X8}");
+        }
+        fixed (int *lineCountPtr = &Brogram.LineCount)
```

Comment:
I'm not particularly proud of my changes to the DevUnit test. Could you clean it up? E.g. it might be worth programmatically checking Brogram.LineCount has the expected value and fail the test if it doesn't. Also, renaming Brogram to something better would be nice.

Reply:
Hopefully fixed.